src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -48,6 +48,9 @@ const INNER_STYLE = {
     minWidth: '100%'
 };
 
+const WIDTH_EPSILON = 0.5;
+const MAX_WIDTH_ITERATIONS = 30;
```

Comment:
30 is a very high number but most of the time it takes 2-6 iterations for the table's column width to settle.